modules/openapi-generator/src/test/resources/3_0/spring/issue_12726.yml

```diff
@@ -0,0 +1,27 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Header validation
+paths:
+  "/headers":
+    options:
+      operationId: headerBeanValidation
+      tags:
+        - headers
+      parameters:
```

Comment:
A test case with Optional will also be interesting.

Reply:
Hi @MelleD , I am not sure I understand, the second header is optional

Reply:
No no i mean the java Optional type. That you generate something like
e.g. `Optional<@Pattern() String>`

Reply:
I would personally not use Optional<> for a request header/parameter

Reply:
@LubomirS somebody else might use Optional here, so please add a test case for that